parsec/core/fs/remote_loader.py

```diff
@@ -798,6 +801,8 @@ async def upload_manifest(
             # Update our cache and retry the request
             self._sequester_services_cache = sequester_services
             return await self.upload_manifest(entry_id, manifest)
+        except VlobSequesterRejectedError as exc:
```

Comment:
Why catching an rebuilding the same error ?

If we want to be more explicit about the fact this exception may bubble up, we can write:
```python
except VlobSequesterRejectedError as exc:
    # You didn't expect me to exist, don't you ?
    raise
```